src/pages/matchmaking.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Matchmaking | Collabify</title>
+  <link rel="stylesheet" href="../styles/matchmaking.css">
+</head>
+<body>
+  <div class="matchmaking-container">
+    <div class="sidebar">
+      <div class="brand-logo">
+        <img src="/img/collabifyLogo.png" alt="Collabify Logo">
+        <h2>Collabify</h2>
+      </div>
+      <ul class="nav-links">
+        <li><a href="profile.html">Profile</a></li>
+        <li><a href="matchmaking.html" class="active">Matchmaking</a></li>
+      </ul>
+    </div>
+
+    <div class="matchmaking-content">
+      <header>
+        <h1>Find Your Match</h1>
+        <p>Search and connect with peers based on skills, learning goals, and projects.</p>
+      </header>
+
+      <section class="search-section">
+        <input 
+          type="text" 
+          id="search-bar" 
+          placeholder="Search for skills, interests, or projects..."
+        />
+        <button id="search-btn">Search</button>
+      </section>
+
+      <section class="filter-section">
+        <h3>Filters</h3>
+        <form id="filters-form">
+          <div class="filter-group">
+            <label for="skills">Skills</label>
+            <select id="skills">
+              <option value="">Select Skill</option>
+              <option value="javascript">JavaScript</option>
+              <option value="python">Python</option>
+              <option value="react">React.js</option>
+              <option value="ai">AI/ML</option>
+            </select>
+          </div>
+          <div class="filter-group">
+            <label for="location">Location</label>
+            <select id="location">
+              <option value="">Select Location</option>
+              <option value="remote">Remote</option>
+              <option value="lagos">Lagos</option>
+              <option value="rivers">Rivers</option>
+            </select>
+          </div>
+          <div class="filter-group">
+            <label for="projects">Project Preferences</label>
```

Comment:
Please explain what you mean by project preference

Reply:
Okay... I'm referring to the stack a consumer or user is interested in.